### PR TITLE
docs: css classes override

### PR DESCRIPTION
**Pull request by odedhutzler**
### Description of the Changes

documentation

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/css-classes-override.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Here's a list of the player classes you can customize:
+
+**Buttons**
+
+|Class Name| Description |
+|--|--|
+| .`playkit-pre-playback-play-button` | The background of the play button, before the player starts playing |
+| `.playkit-icon-play` | The player icon, which is used in the bottom controllers and on the player itself |
+|`.playkit-icon-start-over`| Start over icon, shown at the end of the video|
+| `.playkit-control-button` | The same for all the controllers icon buttons in the bottom control |
```
Comment (AnnaAleksandrowicz) [inline docs/css-classes-override.md]: I don't understand what you mean by "The same for all the controllers icon buttons in the bottom control" - please clarify!

File: docs/css-classes-override.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| `.playkit-seek-bar .playkit-progress-bar .playkit-progress .playkit-scrubber` | Indicates the end of the progress bar. When hovering over the seek bar, it will indicate the current position of the mouse on the seek bar |
+| `.playkit-seek-bar .playkit-progress-bar .playkit-virtual-progress` | When hovering and seeking to a future part of the media, this will show the progress until this point |
+| `.playkit-seek-bar .playkit-progress-bar .playkit-time-preview` | When hovering on the seek bar, it will show the current mouse position time |
+
+**Overlays**
+
+| Class Name | Description |
+|--|--|
+|`.playkit-bottom-bar`| Placeholder of all the bottom controllers|
+|`.playkit-overlay .playkit-overlay-contents`| Overlays container, for advanced captions settings (for example) |
```
Comment (AnnaAleksandrowicz) [inline docs/css-classes-override.md]: Please make sure the change I made here is correct - you wrote "e.g., and that usually should have something that follows...

File: docs/css-classes-override.md (hunk 1 above)
Comment (odedhutzler) [reply docs/css-classes-override.md]: "a class attached to all the buttons in the control container, a change here will affect all the buttons"

Comment (AnnaAleksandrowicz) [reply docs/css-classes-override.md]: So let's change this to "This is a class that is attached to all the buttons in the control container; a change here will affect all of the buttons."

OK

**Pull request merged**